shared/js/participantStore.js

```diff
@@ -41,14 +41,14 @@ loop.store.ParticipantStore = function() {
 
     updatedParticipant(actionData) {
       this._updateParticipantData(actionData.userId, {
-        name: actionData.participantName
+        participantName: actionData.participantName
       });
     },
 
     updatedPresence(actionData) {
       this._updateParticipantData(actionData.userId, {
         isHere: actionData.isHere,
-        localPingTime: Date.now() - actionData.pingedAgo
+        localPingTime: actionData.pingedAgo
```

Comment:
`Date.now() - 1000` should result in a number like.. 1465202237419. And a roughly 30 seconds later when I do `Date.now() - 1465202237419` I get 32931.